docs/core-concepts/testing.md

```diff
@@ -1,19 +1,63 @@
 
-As a short-cut you can also use their demo repository that hosts a Docker packaged EVerest image. [See here for Github Repo](https://github.com/EVerest/everest-demo)
+# Running Everest
+In order to alleviate some of the complexities that may arise when starting Everest, we have created
+some helpful commands that should help in getting the Everest charger simulator running locally and targeting
+CitrineOS.
 
-If you want to run EVerest on the side while developing and making changes, you can follow the steps below.
+You will notice in `/Server/everest` directory the new files created to support running Everest within Docker.
+In addition, we created some helpful NPM commands:
+
+- `npm run start-everest`
+- and
+- `npm run start-everest-windows`
+
+Both of which in essence do the same thing which is to trigger the `docker compose up` command from within
+the `/Server/everest` directory so that it can pick up the `Dockerfile` and the `docker-compose.yml` files:
+
+```shell
+EVEREST_IMAGE_TAG=0.0.16 EVEREST_TARGET_URL=ws://host.docker.internal:8081/cp001 docker-compose up
+```
+
+You will notice that there are two args that are configurable:
+
+- `EVEREST_IMAGE_TAG` - determines the image tag that will be used for the Everest image (ghcr.io/everest/everest-demo/manager)
+- `EVEREST_TARGET_URL` - the URL that Everest will point to. Defaulting to `host.docker.internal` assuming CitrineOS will run on
+  same machine, since `localhost` won't work within Docker
+
+After running `npm run start-everest` (or the Windows alternative), you should see 3 running Everest containers
+and the `manager` container should have the appropriate Everest logs.
+
+### Everest UI
+Now that the 3 containers are running in Docker, you should be able to navigate to `[localhost|ip]:1880/ui/` to view
+the Everest simulator UI. There, you should be able to simulate the pause/resume and plug/unplug events among others.
+
+### Everest NodeRed
+You can also view the Everest NodeRed UI `[localhost|ip]:1880/`, but it is not advisable to make any adjustments here
+unless you have a good understanding of this configuration.
+
+### Viewing OCPP logs in Everest
+To view the OCPP logs in Everest, we have utilized Node `http-server`, which you will see being initialized
+in the Dockerfile. We initialize a simple HTTP server on port `8888` and expose this port so that it is
+mapped in the compose file allowing you to navigate to `localhost:8888`. This HTTP server is configured to
+serve the contents of the `/tmp/everest_ocpp_logs` which is where Everest stores the OCPP logs in the
+Docker container. Conveniently, the logs are in HTML format, so we can easily view them in the browser.
+
+# Running Everest Manually
+You can also use their demo repository that hosts a Docker packaged EVerest image. [See here for Github Repo](https://github.com/EVerest/everest-demo)
+
+To get EVerest running on the side while developing and making changes, you can follow the steps below.
```

Comment:
Minor: inconsistent with usage of "EVerest" vs "Everest"

Reply:
adjusted 👍 thank you 🙇